plugins/alignments/src/LinearAlignmentsDisplay/models/model.tsx

```diff
@@ -361,10 +361,13 @@ function stateModelFactory(
          * #method
          */
         trackMenuItems(): MenuItem[] {
+          if (!self.PileupDisplay) {
+            return []
+          }
           const extra = getLowerPanelDisplays(pluginManager).map(d => ({
             type: 'radio',
             label: d.displayName,
-            checked: d.name === self.PileupDisplay.type,
+            checked: d.name === self.PileupDisplay ?? false,
```

Comment:
i think that you can keep this line the same since the above check for undefined ensures it is not undefined